src/create_disk.sh

```diff
@@ -79,18 +88,56 @@ sgdisk -Z $disk \
 	-n 4:0:0     -c 4:root       -t 4:4F68BCE3-E8CD-4DB1-96E7-FBCAF984B709
 sgdisk -p "$disk"
 
-udevadm trigger
-udevadm settle
+udevtrig
+
+root_dev="${disk}4"
+if [ -n "${luks_rootfs}"  ]; then
+    root_dev=/dev/mapper/crypt_root
+    sgdisk -c 4:luk_root "${disk}"
+
+    touch tmp.key
+    # Create the LUKS partition using the null_cipher and a sentinal
+    # UUID similiar to the one used by coreos-gpt-setup. This is used
+    # by ignition-dracut-reecrypt.
```

Comment:
Is the code for this online somewhere yet?  Are you planning it to be a PR to the ignition-dracut repo or a separate one?